README.md

```diff
@@ -48,11 +48,10 @@ For F# Compiler on OSX and Linux (see .travis.yml for build steps)
 
 For Visual F# IDE Tools 4.1 development (Windows)
 
-- [Visual Studio 15](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx)
-  - Select Visual C++ / Common Tools For Visual C++
-  - Visual F#
-  - Visual Studio Extensibility Tools
-- [Visual Studio 15 SDK](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx)
+- [Visual Studio 2017 RC](https://www.visualstudio.com/vs/visual-studio-2017-rc/#downloadvs)
+  - Under the "Windows" workloads, select ".NET desktop development".
+    - Select "F# language suport" under the optional components.
```

Comment:
I'm not aware of any such quotes, but you're certainly free to raise that issue with appropriate folks.  We have no say in this, though.

Something to note is that you could technology follow these same instructions for the "Web" workload, and be just as successful (as it's the .NET Web workload, and F# is also in the same place as .NET Desktop)_.  It's also present in Data Storage and Processing, Xamarin, and Data Science (and the Individual Components tab, but all components are there).